src/nlp-visual-editor/nlp-visual-editor.jsx

```diff
@@ -100,13 +104,15 @@ class VisualEditor extends React.Component {
       errorMessage: undefined,
       languageSelectModal: false,
       showSettings: false,
-      editorSettings: localStorage.getItem('nlpEditorSettings') ? JSON.parse(localStorage.getItem('nlpEditorSettings')) : {
-        moduleName: 'elyraNLPCanvas',
-        language: 'en'
-      }
+      editorSettings: localStorage.getItem('nlpEditorSettings')
+        ? JSON.parse(localStorage.getItem('nlpEditorSettings'))
+        : {
+            moduleName: 'elyraNLPCanvas',
+            language: 'en',
```

Comment:
Would it be possible to make 'en' a constant, as it is used repeatedly in multiple places? It could be called "default_language" or something like this, and replace all usages of `en` with the `default_language` constant.